frontend/awx/resources/templates/TemplatePage/steps/TemplateLaunchReviewStep.tsx

```diff
@@ -137,7 +137,7 @@ export default function TemplateLaunchReviewStep(props: { template: JobTemplate
   );
 }
 
-function CredentialDetail({ credentialID }: { credentialID: number }) {
+export function CredentialDetail({ credentialID }: { credentialID: number }) {
```

Comment:
Do we need to export this?

Reply:
I expect that this type of credential function will be needed elsewhere.  I would be on board with moving into a generic component directory somewhere.